README.md

```diff
@@ -72,6 +73,31 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-check/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+### What features does `cargo-semver-checks` enable in the tested crates?
+
+By default, crates are built with all features excluding heuristically selected features that usually break semver. More precisely, features named `unstable`, `nightly`, `bench`, `no_std` or starting with `__` are disabled.
+
+This behaviour can be overriden. Checked feature set can be changed to:
+- *all* the features, selected with `--all-features`,
+- only the crate's default features, selected with `--default-features`,
+- empty set, selected with `--only-explicit-features`.
+
+Additionally, features can be enabled one-by-one, using flags `--features`, `--baseline-features` and `--current-features`.
+
+For example, consider crate [serde](https://github.com/serde-rs/serde), with the following features (per v1.0.163):
+- `std` - the only crate's default feature,
+- `alloc`, `derive`, `rc` - optional features,
+- `unstable` - a feature that possibly breaks semver.
+
+| used flags | selected feature set |
+|--|--|
+| none | `std`, `alloc`, `derive`, `rc`  |
+| `--features unstable` | `std`, `alloc`, `derive`, `rc`, `unstable` |
+| `--all-features` | `std`, `alloc`, `derive`, `rc`, `unstable` |
+| `--default-features` | `std` |
+| `--default-features --features derive` | `std`, `derive` |
+| `--only-explicit-features --features unstable` | `unstable` |
```

Comment:
This is much clearer already.

Have you thought about possibly adding an "explanation" column to the right?

For example:
- In the `none` row it could say: "Feature `unstable` is excluded by the default heuristic."
- In the next row it could say: "The flag explicitly adds `unstable` to the heuristic's selections."
etc.

Reply:
I think this is a good idea, unfortunately now there are line breaks in the flags, but I don't think that is a big issue.

Reply:
yeah, it's unfortunate but definitely the lesser problem, this is fine as-is